### PR TITLE
Added en, ru, ua localization to Team page and its components

### PR DESCRIPTION
**Pull request by YrPryhoda**

### CODE REVIEW

File: client/src/components/TeamWorkedProjects/index.tsx [hunk 1]
```diff
@@ -42,11 +42,7 @@ const TeamWorkedProjects = ({ projects }: Props) => {
...
 							<div className={styles.worked_block}>
 								<Header as="h4">{t('no_works')}</Header>
-								<p className={styles.p_descr}>
-									{t(
-										'There are no projects which have been started yet. When it&apos;ll happen you can get quick access to one you need',
-									)}
-								</p>
+								<p className={styles.p_descr}>{t('there_are_no_projects')}</p>
 							</div>
 						</>
```
Comment (R-Mielamud) [inline client/src/components/TeamWorkedProjects/index.tsx:47]: there_are_no_projects_which_have_been_started_yet

File: client/src/translations/ua.ts [hunk 2]
```diff
@@ -268,4 +268,30 @@ export const ua = {
...
+	link_to: 'Посилання на',
+	will_be_deleted_from_section: 'буде видалено з розділу посилань на матеріали команди',
+	project_key: 'Ідентифікатор проекту',
+	there_are_no_projects: 'У вас ще немає проектів, які були розпочаті. Коли це станеться, ви зможете швидко отримати доступ до потрібної Вам роботи',
+	team_owner: 'Творець команды',
+	you_are_the_first: 'Ви перший і поки що єдиний учасник команди',
+
+
+
 };
```
Comment (R-Mielamud) [inline client/src/translations/ua.ts:297]: Remove space before }

File: client/src/translations/ru.ts [hunk 3]
```diff
@@ -273,4 +273,28 @@ export const ru = {
...
+	small_description: 'Небольшое описание',
+	delete_link: 'Удалить ссылку на материал',
+	link_to: 'Ссылка на',
+	will_be_deleted_from_section: 'будет удалена из раздела материалов команды',
+	project_key: 'Идентификатор проекта',
+	there_are_no_projects: 'У вас еще нет проектов, которые были начаты. Когда это произойдет, вы сможете быстро получить доступ к нужной Вам работе',
+	team_owner: 'Создатель команды',
+	you_are_the_first: 'Вы первый и пока что единственный участник команды',
+
 };
```
Comment (R-Mielamud) [inline client/src/translations/ru.ts:300]: Remove space before }

File: client/src/translations/en.ts [hunk 4]
```diff
@@ -303,4 +303,28 @@ export const en = {
...
+	small_description: 'Small description',
+	delete_link: 'Delete this link',
+	link_to: 'Link to',
+	will_be_deleted_from_section: 'will be deleted from team links section',
+	project_key: 'Project key',
+	there_are_no_projects: 'There are no projects which have been started yet. When it&apos;ll happen you can get quick access to one you need',
+	team_owner: 'Team owner',
+	you_are_the_first:'You are the first member of this team',
+
 };
```
Comment (R-Mielamud) [inline client/src/translations/en.ts:330]: Remove space before }

**Review by R-Mielamud — CHANGES_REQUESTED** [review]

File: client/src/components/TeamDevsCard/index.tsx [hunk 5]
```diff
@@ -55,6 +55,7 @@ const TeamDevsCard = ({
 			</Card.Content>
 			<Card.Content>
 				<textarea
+					placeholder={t('add_some_description')}
 					disabled={lockEditFields}
 					onChange={(e) => setTeamDescription(e.target.value)}
 					defaultValue={teamDescription}
```
[the remark this answers is not in the document]
Comment (YrPryhoda) [reply client/src/components/TeamDevsCard/index.tsx]: it's not difficult. ok

File: client/src/components/TeamLinks/createLink.tsx [hunk 6]
```diff
@@ -37,35 +40,35 @@ const CreateLink = ({ onClose, currentLink, onConfirm }: Props) => {
...
 					<Form.Field>
-						<label>Title</label>
+						<label>{t('title')}</label>
 						<input
 							value={newLink.name}
 							onChange={(e) => onChange(e)}
-							placeholder="For example: My first project"
+							placeholder={t('for_example ')}
 							name="name"
 						/>
```
[the remark this answers is not in the document]
Comment (YrPryhoda) [reply client/src/components/TeamLinks/createLink.tsx]: renamed

File: client/src/components/TeamWorkedProjects/index.tsx (hunk 1 above)
Comment (YrPryhoda) [reply client/src/components/TeamWorkedProjects/index.tsx]: corrected

**Review by R-Mielamud — APPROVED** [review]
All fine

**Pull request merged**